sources/packages/api/src/database/entities/education-program.model.ts

```diff
@@ -11,9 +11,18 @@ import { ColumnNames, TableNames } from "../constant";
 import { RecordDataModel } from "./record.model";
 import { Institution, Note, User } from ".";
 import { ProgramIntensity } from "./program-intensity.type";
-import { ApprovalStatus } from "../../services/education-program/constants";
 import { dateOnlyTransformer } from "../transformers/date-only.transformer";
 
+/**
+ * Represents the approval status for a program also stored
+ * on column approval_status on table education_programs.
+ */
+export enum ProgramStatus {
```

Comment:
Can we please have this declared in the same way as the majority of the other enums in separated files like `some-enum-status.type.ts`?